packages/core/core/src/factories/useCartFactory.ts

```diff
@@ -113,6 +113,21 @@ export const useCartFactory = <CART, CART_ITEM, PRODUCT, COUPON>(
       }
     };
 
+    const refresh = async ({ customQuery } = { customQuery: undefined }) => {
+      Logger.debug('useCart.refresh');
+
+      try {
+        loading.value = true;
+        cart.value = await _factoryParams.load({ customQuery });
+        error.value.load = null;
+      } catch (err) {
+        error.value.load = err;
```

Comment:
`error.value.refresh` should be supported imo

Reply:
If you decide to add it, do not forget to add it at the very top of the file inside ` const error: Ref<UseCartErrors> = sharedRef({` object

Reply:
please notice that load calls refresh - the difference is only in the side effects - load can load the cart only once, while refresh loads it always

Reply:
So we want to have `error.value.refresh` not `.load`, wdyt?